medical_patient_disease/views/medical_patient_disease_view.xml

```diff
@@ -0,0 +1,199 @@
+                <separator />
+                <filter name="is_infectious"
+                        string="Is Infectious"
+                        domain="[('is_infectious', '=', True)]" />
+                <filter name="is_not_infectious"
+                        string="Not Infectious"
+                        domain="[('is_infectious', '=', False)]" />
+                <newline />
+                <group expand="0" string="Group By...">
+                    <filter string="Healed"
+                            domain="[]"
+                            context="{'group_by':'healed_date'}" />
+                    <filter string="Disease"
+                            domain="[]"
+                            context="{'group_by':'pathology_id'}" />
+                    <filter string="Allergic Disease"
+                            domain="[]"
+                            context="{'group_by':'is_allergy'}" />
+                    <filter string="Physician"
+                            domain="[]"
+                            context="{'group_by':'physician_id'}" />
+                    <filter string="Pregnancy warning"
+                            domain="[]"
+                            context="{'group_by':'is_pregnant'}" />
+                    <filter string="Contracted in pregnancy week #"
+                            domain="[]"
+                            context="{'group_by':'weeks_of_pregnancy'}" />
+                    <filter string="Currently on Treatment"
+                            domain="[]"
+                            context="{'group_by':'is_on_treatment'}" />
+                    <filter string="Date of Diagnosis"
+                            domain="[]"
+                            context="{'group_by':'diagnosed_date'}" />
+                    <filter string="Infectious Disease"
+                            domain="[]"
+                            context="{'group_by':'is_infectious'}" />
+                    <filter string="Patient"
+                            domain="[]"
+                            context="{'group_by':'patient_id'}" />
+                    <filter string="Age when diagnosed"
+                            domain="[]"
+                            context="{'group_by':'age'}" />
+                    <filter string="Start"
+                            domain="[]"
+                            context="{'group_by':'treatment_start_date'}" />
+                    <filter string="State"
+                            domain="[]"
+                            context="{'group_by':'state'}" />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="medical_patient_disease_view_tree" model="ir.ui.view">
+        <field name="name">medical.patient.disease.tree</field>
+        <field name="model">medical.patient.disease</field>
+        <field name="arch" type="xml">
+            <tree string="Medical Patient Disease">
```

Comment:
Just as a note I removed a lot of the fields from the tree view here as it was really crowded in the view. These were the fields I left in, which I thought were of importance. However I can add them back either as visible or invisible if needed, or change which fields I left here.